tasks/libs/ciproviders/github_api.py

```diff
@@ -34,6 +34,13 @@ def repo(self):
 
         return self._repository
 
+    def raw_request(self, url):
```

Comment:
I don't think we need this, I give some code examples below

Reply:
Ok, I actually think it is better as it avoids that extra method added to the api class. It is still slower and maybe harder to follow but at least in one place. I had to tweak your suggestion a little but tested it and it works good. Thanks!